marconi-core/src/Marconi/Core/Type.hs

```diff
@@ -27,6 +27,7 @@ import Data.Data (Typeable)
 import Data.List.NonEmpty (NonEmpty)
 import Data.Text (Text)
 import GHC.Generics (Generic)
+import Marconi.Core.TracedStorable (HasPoint (getPoint))
```

Comment:
This comes from the old implementation and shouldn't been used (maybe we should add a `Legacy` to the module names). To remove it, you probably need to remove one abstraction level to `withStability`: `g event -> Timed (Point event) event`

Reply:
Ah, I just noticed this and resolved it by sticking a copy of the typeclass into `Core.Type`. What do you reckon?